src/firestore/util.ts

```diff
@@ -127,6 +160,9 @@ export const initializeFirebaseProject = async (
   name: string,
   authPersistence = AuthPersistence.session,
   markRawConfig: MarkRawConfig = {},
+  // App Check reCAPTCHA site keys and debug token
+  siteKey = '',
+  debugToken = '',
```

Comment:
These params are now encapsulated in the `config`, right? It seems like we are passing them twice. Can we just rely on them being nested inside of the `config`?